src/components/Common/Filters/FavorisFilter.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <ul class="list-group">
+    <FavoriFilterIteam
+      v-for="(favori, i) in favoris"
+      :key="i"
+      :index="index"
+      :collection="collection"
+      :favori="favori"
+      @favoris-delete="onFavoriDelete"
+    />
+  </ul>
+</template>
+
+<script>
+import * as filterManager from '../../../services/filterManager'
+import FavoriFilterIteam from './FavoriFilterIteam'
+
+export default {
+  name: 'FavorisFilter',
```

Comment:
```suggestion
  name: 'FavoriteFilters',
```

(apply this to the file name)